src/apps/loyalty/auth.ts

```diff
@@ -0,0 +1,7 @@
+export function login(email, pass, cb) {
+  cb(true)
```

Comment:
For actions like this, we should use `redux`, but that's a whole different conversation. For now I'm 👍 with this.